bin/install

```diff
@@ -46,6 +46,9 @@ function install_rabbitmq {
 		_semver=${_version#v}
 	elif [[ ${version} =~ "v".* ]]; then
 		_semver=${version#v}
+	elif [[ ${version} =~ ^[0-9]+\.[0-9]+\.[0-9]+(.*)?$ ]]; then
+		version="v${version}"
+		_semver=${version#v}
```

Comment:
it's a little we could do this step outside of the conditional since it's always being done, but that might make the middle conditional a little awkward since bash doesn't like it when you have a conditional with an empty body I don't think, so it may be worth keeping things the way they are for that reason.

Up to you.